_topic_maps/_topic_map.yml

```diff
@@ -2067,6 +2085,8 @@ Topics:
     File: creating-machineset-gcp
   - Name: Creating a compute machine set on IBM Cloud
     File: creating-machineset-ibm-cloud
+  - Name: Creating a machine set on IBM Power Virtual Server
```

Comment:
```suggestion
  - Name: Creating a compute machine set on IBM Power Virtual Server
```

Reply:
Thanks for the review comments @jeana-redhat 
I have incorporated the suggested changes.